packages/app/src/app/pages/common/Modals/LiveSessionConfirm/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Button, Stack, Text } from '@codesandbox/components';
+import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
+import track from '@codesandbox/common/lib/utils/analytics';
+
+import { useWorkspaceAuthorization } from 'app/hooks/useWorkspaceAuthorization';
+import { useCreateCheckout } from 'app/hooks';
+import { useWorkspaceSubscription } from 'app/hooks/useWorkspaceSubscription';
+import { useActions, useAppState } from 'app/overmind';
+import { Alert } from '../Common/Alert';
+
+export const LiveSessionConfirm: React.FC = () => {
+  const {
+    activeTeam,
+    editor: {
+      currentSandbox: { id },
+    },
+  } = useAppState();
+  const { modalClosed, live } = useActions();
+  const [checkout, createCheckout] = useCreateCheckout();
+  const { isAdmin } = useWorkspaceAuthorization();
+  const { isEligibleForTrial } = useWorkspaceSubscription();
+
+  React.useEffect(() => {
+    track('Live Session - notice seen', {
+      codesandbox: 'V1',
+      event_source: 'UI',
+    });
+  }, []);
+
+  return (
+    <Alert title="After February 20, Live Sessions will become a Pro feature.">
+      {checkout.status === 'error' && (
+        <Text marginBottom={8} variant="danger" size={12}>
+          An error ocurred while trying to load the checkout. Please try again.
+        </Text>
+      )}
+      <Stack gap={2} align="center" justify="flex-end">
+        <Button
+          onClick={() => {
+            live.createLiveClicked(id);
+            modalClosed();
+          }}
+          variant="link"
+          autoWidth
+        >
+          Go Live now
+        </Button>
+        {isAdmin ? (
+          <Button
+            loading={checkout.status === 'loading'}
+            variant="primary"
+            onClick={() => {
+              track('Live Session - upgrade clicked', {
+                codesandbox: 'V1',
+                event_source: 'UI',
+              });
+
+              createCheckout({
+                team_id: activeTeam,
+                recurring_interval: 'month',
+                success_path: sandboxUrl({ id }),
+                cancel_path: sandboxUrl({ id }),
+              });
+            }}
+            autoWidth
+          >
+            <Text>
+              {isEligibleForTrial ? 'Start free trial' : 'Upgrade to Pro'}
+            </Text>
+          </Button>
+        ) : (
+          <Button
+            as="a"
+            href="/pricing"
```

Comment:
@necoline in the designs, it says to go to `/pro` but that page isn't useful for non-admins. Instead, I linked to `/pricing`, is there another place I should link to?

Reply:
In effect, the non-admins will see the same information on `/pricing` as they would on `/pro` the difference is that I think pro is a more useful link for the non-admin to send to the admin if they want to advocate for the upgrade

Reply:
Adjusted in [64972d5](https://github.com/codesandbox/codesandbox-client/pull/7413/commits/64972d57dca3d2c47717b697f6595ca4759390d3)